sample/src/main/java/com/clevertap/demo/MyApplication.kt

```diff
@@ -147,6 +146,12 @@ class MyApplication : MultiDexApplication(), CTPushNotificationListener, Activit
                 //.penaltyDeath()
                 .build()
         )*/
+   /*     StrictMode.setVmPolicy(
```

Comment:
Do we need this commented-out code ?

Reply:
Yes, it's a toggle for testing purposes.

Reply:
Ok, maybe a comment above the block explaining what it's for would be useful.

Reply:
Fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/c301c4cd97fefe36a6e4e6e8ef3c84e7c1f4baf9